extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -16,8 +16,11 @@
 
 import '../amp-ad-exit';
 import * as sinon from 'sinon';
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.